### ISSUE TITLE
[JOSS Review] Add contribution guidelines

### ISSUE THREAD

**Issue by blsqr** [simpeg/aurora#332]
Hi @kkappler and co-authors,

I'm here as one of the reviewers for your [submission to JOSS](https://github.com/openjournals/joss-reviews/issues/6832) and will be opening a few issues as part of the review process. Some of them will be suggestions or questions, others I would deem required to address review criteria. I am happy to discuss anything or even assist in implementation, if this is something you would consider.

This issue pertains to a requirement for JOSS publications of having clear community and contribution guidelines:
> Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support?

I could not find any information regarding this in your repository or the documentation. Did I miss anything?

If not, this requirement could be addressed with a "Contributing to aurora" section in the README and the documentation; or a separate `CONTRIBUTING.md` file at the top-level of the repository (linked to from the README and docs), which I feel is an established way of communicating contribution guidelines.

For a collection of resources around open source contributions, I find the following quite useful: https://github.com/github/opensource.guide … but of course you would need to make your own decisions regarding how best to do this.

**Comment by diehlpk** [comment]
Hi @kkappler could you please have a look?

**Comment by kkappler** [comment]
Hi @blsqr, thanks for opening this issue.

Aurora is hosted by the [simpeg](https://simpeg.xyz/) project, and they have a fairly extensive [contributing guidelines](https://docs.simpeg.xyz/latest/content/getting_started/contributing/index.html).  

To address this issue, I added a minimal CONTRIBUTING.rst borrowing some text from the opensourceguide link you shared, and then added a link to the simpeg guidelines in the .rst.  


There is a [pull request](https://github.com/simpeg/aurora/pull/334) staged to merge these changes into JOSS branch.  I tried to add @blsqr as a reviewer on the PR, but I think I would need to add you formally to the project -- Happy to do that if you request.
